reference-architectures/cloud_deploy_flow/CloudBuild/buildCloudRun.yaml

```diff
@@ -0,0 +1,20 @@
+steps:
```

Comment:
To increase build reproducibility, consider using fixed container image tags, or even hashes